storybook/stories/support/StoryWrapper.js

```diff
@@ -33,20 +34,36 @@ const locales = {
 };
 const localeNames = keys(locales);
 
+const operatingSystems = {
+  Windows: 'windows',
+  Linux: 'linux',
+  Mac: 'mac',
+};
+
+const osMinWindowHeights = {
```

Comment:
@DeeJayElly are these numbers going to be correct once #1485 is merged?

Reply:
@DeeJayElly could you add a comment, explaining why there are different numbers for each OS (e.g. inner menu, etc)